code/modules/mob/living/simple_animal/hostile/nanotrasen.dm

```diff
@@ -0,0 +1,67 @@
+/mob/living/simple_animal/hostile/nanotrasen
+	name = "Nanotrasen Private Security Officer"
+	desc = "An officer part of Nanotrasen's private security force, he seems rather unpleased to meet you."
+	icon_state = "nanotrasen"
+	icon_living = "nanotrasen"
+	icon_dead = null
+	icon_gib = "syndicate_gib"
+	speak_chance = 25
+	turns_per_move = 5
+	response_help = "pokes"
+	response_disarm = "shoves"
+	response_harm = "hits"
+	speed = 0
+	stat_attack = 1
+	robust_searching = 1
+	maxHealth = 100
+	health = 100
+	harm_intent_damage = 5
+	melee_damage_lower = 10
+	melee_damage_upper = 15
+	attacktext = "punches"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	a_intent = "harm"
+	var/corpse = /obj/effect/landmark/corpse/nanotrasensoldier
```

Comment:
Might be a problem with making this a `corpse` instead of a `mobcorpse`, because ghosts could then click on the dead nanotrasen officer and play as them. Is this intended?